lib/cloud_controller/metrics/prometheus_updater.rb

```diff
@@ -35,7 +36,15 @@ def self.allow_pid_label
       { type: :gauge, name: :cc_running_tasks_total, docstring: 'Total running tasks', aggregation: :most_recent },
       { type: :gauge, name: :cc_running_tasks_memory_bytes, docstring: 'Total memory consumed by running tasks', aggregation: :most_recent },
       { type: :gauge, name: :cc_users_total, docstring: 'Number of users', aggregation: :most_recent },
-      { type: :gauge, name: :cc_deployments_in_progress_total, docstring: 'Number of in progress deployments', aggregation: :most_recent }
+      { type: :gauge, name: :cc_deployments_in_progress_total, docstring: 'Number of in progress deployments', aggregation: :most_recent },
+      { type: :gauge, name: :cc_acquired_db_connections_total, labels: %i[process_type], docstring: 'Number of acquired DB connections' },
+      { type: :histogram, name: :cc_db_connection_hold_duration_seconds, docstring: 'Durations of connections being hold by threads', buckets: CONNECTION_HOLD_DURATION_BUCKETS },
```

Comment:
*nit/suggestion* Should this be past tesnse? e.g. `cc_db_connection_held_duration_seconds` as we only emit this metric on release

Reply:
Hmm somehow `hold_duration` sounds more intuitive to me as the action of holding connections is being measured. Don't have a strong opinion here though.

Reply:
👍  no strong opinions either - will leave it up to you.